libs/src/services/web3Manager/index.js

```diff
@@ -101,9 +101,8 @@ class Web3Manager {
    * @param {string} data
    */
   async sign (data) {
-    const account = this.getWalletAddress()
-
     if (this.useExternalWeb3) {
+      const account = this.getWalletAddress()
       if (this.isServer) {
         return this.web3.eth.sign(this.web3.utils.fromUtf8(data), account)
       } else {
```

Comment:
there's also another change in the else clause https://github.com/AudiusProject/audius-protocol/pull/516/files#diff-f08bd6bbe58b74a2dd5c0343aa19f400R111 we could move over

Reply:
functionally same, all good

Reply:
:) i didn't think it was necessary so i didnt move it over!